afs/afs-ws/afs-ws-server/src/main/java/com/powsybl/afs/ws/server/AppStorageServer.java

```diff
@@ -340,7 +340,7 @@ public Response isWritable(@ApiParam(value = "File system name") @PathParam("fil
 
     @GET
     @Produces(MediaType.TEXT_PLAIN)
-    @Path("fileSystems/{fileSystemName}/nodes/{nodeId}/consistent")
+    @Path("fileSystems/{fileSystemName}/nodes/{nodeId}/isConsistent")
```

Comment:
isConsistent is a bad REST design as it embed the verb. It has to be design in term of resource and not action.